client/decrypt/decrypt.go

```diff
@@ -0,0 +1,102 @@
+package decrypt
```

Comment:
I'm not sure if this name would be good.

`crypto/decrypt`?
`xxx/decrypt`?
difficult!

Reply:
Yep it looks good to me too!